lesson5.html

```diff
@@ -0,0 +1,45 @@
+<html>
+    <head>
+        <title>Mouse leave and over event</title>
+        <link rel="stylesheet" type="text/css" href="style5.css">
+    </head>
+    <body>
+        <div id ="newBtn"></div>
+        <div class="btn-postn"><button id ="awsome">Just-Awsome</button></div>
+        
+        <script>
+            let awsmBttnCmp = document.getElementById("awsome");
+            let newCmp = document.getElementById("newBtn");
+            const label = "Im Awsome" ;
+             function onClick(event){
+                 console.log("Hey!! Be positive");
+             }
+             function mouseOverHandler (event){
+                 console.log("You are here");
+                 awsmBttnCmp.innerText = "Be Awsm" ;
+                 
+                 awsmBttnCmp.classList.add("button-shape");
+                 let bttnNew = document.createElement("button");
+                 awsmBttnCmp.classList.add("button-hover");
+                 awsmBttnCmp.classList.remove("button-original");
+                 bttnNew.innerText = "Stay Calm" ;
+                 newCmp.appendChild(bttnNew); 
+                 p.style.display="block";
```

Comment:
What us p in this case? There are some errors in the console, that elements are not being created before called.
To complete the assignment you can actually remove functionality that creates new button on click (leave just console message on click) - this would be the quickest fix for you I guess.